exporter/importables.go

```diff
@@ -527,9 +527,15 @@ var resourcesMap map[string]importable = map[string]importable{
 					log.Printf("[INFO] Group %s doesn't match %s filter", g.DisplayName, ic.match)
 					continue
 				}
+				hclblockMode := "resource"
+				genDataBlock := map[string]bool{"admins": true, "users": true}
+				if genDataBlock[g.DisplayName] {
+					hclblockMode = "data"
+				}
```

Comment:
This splits logic into two pieces that a harder to handle. It's better to do everything in the `Import` function (see comment there)

Reply:
Yes. The logic I feel - should be split
List() is for discovery. You mark all your special groups there. Right now it's admins and users.. in the future it can be others.
By the time you get to import you're not creating exceptions - just handling data blocks differently. 
Perhaps in the future we should create config files with maps in them so that instead of rolling out code changes we roll out new config files

Reply:
The problem is that emit of the `databricks_group` not necessary happens via `List` - if you check the code, it happens from inside group's Import, from `databricks_user`, from `databricks_permission`, and maybe some other places.  

This is necessary because emit could be triggered by another object, for example, if you do a selective export of a single cluster, then you emit a cluster, Import of a cluster emits a permission, and Import of permission emits group that is then handled in Import